dist-legacy/src/distribution/resources/modules/system/layers/base/org/jboss/as/product/wildfly-core/dir/META-INF/MANIFEST.MF

```diff
@@ -1,2 +1,11 @@
 JBoss-Project-Release-Name: WildFly Core
 JBoss-Product-Release-Version: ${project.version}
+JBoss-Project-Banner: \n
+  _    _ _ _     _______ _              _____                \n
+ | |  | (_) |   | |  ___| |            /  __ \               \n
+ | |  | |_| | __| | |_  | |_   _ ______| /  \/ ___  _ __ ___ \n
+ | |/\| | | |/ _` |  _| | | | | |______| |    / _ \| '__/ _ \\n
+ \  /\  / | | (_| | |   | | |_| |      | \__/\ (_) | | |  __/\n
+  \/  \/|_|_|\__,_\_|   |_|\__, |       \____/\___/|_|  \___|\n
+                            __/ |                            \n
+                           |___/
```

Comment:
I like the idea but I don't like that specific banner (a bit too high) :) (text should be "WildFly Core", not "WildFly-Core" btw).
TGIF, I played with http://patorjk.com/software/taag/#p=testall&f=Ogre&t=WildFly%20Core and found these ones nice:

```

 _  _  __  __    ____  ____  __    _  _     ___  __  ____  ____ 
/ )( \(  )(  )  (    \(  __)(  )  ( \/ )   / __)/  \(  _ \(  __)
\ /\ / )( / (_/\ ) D ( ) _) / (_/\ )  /   ( (__(  O ))   / ) _) 
(_/\_)(__)\____/(____/(__)  \____/(__/     \___)\__/(__\_)(____)


 _       ___ __    __________         ______              
| |     / (_) /___/ / ____/ /_  __   / ____/___  ________ 
| | /| / / / / __  / /_  / / / / /  / /   / __ \/ ___/ _ \
| |/ |/ / / / /_/ / __/ / / /_/ /  / /___/ /_/ / /  /  __/
|__/|__/_/_/\__,_/_/   /_/\__, /   \____/\____/_/   \___/ 
                         /____/                           
  _      ___ __   ________       _____            
 | | /| / (_) /__/ / __/ /_ __  / ___/__  _______ 
 | |/ |/ / / / _  / _// / // / / /__/ _ \/ __/ -_)
 |__/|__/_/_/\_,_/_/ /_/\_, /  \___/\___/_/  \__/ 
                       /___/                      

 __        ___ _     _ _____ _          ____               
 \ \      / (_) | __| |  ___| |_   _   / ___|___  _ __ ___ 
  \ \ /\ / /| | |/ _` | |_  | | | | | | |   / _ \| '__/ _ \
   \ V  V / | | | (_| |  _| | | |_| | | |__| (_) | | |  __/
    \_/\_/  |_|_|\__,_|_|   |_|\__, |  \____\___/|_|  \___|
                               |___/                       

```

Reply:
ok it depends on the font size :)